Add kserve install instrunctions (#2192)
kubeflow/manifests: README.md

kustomize build apps/kfserving/upstream/overlays/kubeflow | kubectl apply -f -
```

> If you don't need KFServing and want to use kserve as integrated mode not stand-alone modee, edit kustomization.yaml of kserve.

Reviewer: ```suggestion
If you don't need KFServing and want to use kserve in an integrated mode and not in standalone mode, then edit the kustomization.yaml of kserve.
```
